src/app/Components/doctor-page/gallery/gallery.component.html

```diff
@@ -3,13 +3,13 @@
     <mat-icon>keyboard_arrow_left</mat-icon>
   </button>
   <div id="slider" [style.width]="amountOfPicsOnScreen * widthOf1Image + 'px'">
-    <div id="img-line" [style.width]="images.length * widthOf1Image + 'px'" [style.left]="startIndex * widthOf1Image + 'px'">
+    <!--<div id="img-line" [style.width]="images.length * widthOf1Image + 'px'" [style.left]="startIndex * widthOf1Image + 'px'">
```

Comment:
Remove commented area or explain.

Reply:
no no no. This is the realization of Andrew's avatars display. Since we have no avatars, we have many mistakes, so I commented. But when avatars are going to be there will only have to be commented on it